docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -197,7 +197,26 @@ implementation is used:
   - Set to `false` to disable in-memory caching of metadata files on the 
     coordinator. This cache is not used when `fs.cache.enabled` is set to true.
   - `true`
-
+* - `iceberg.expire_snapshots.min-retention`
+  -  Minimal retention period for `expire_snapshot` procedure. Equivalent
+     session property is `expire_snapshots_min_retention`.
+  -  `7 DAYS` 
+* - `iceberg.remove_orphan_files.min-retention`
+  -  Minimal retention period for `remove_orphan_files` procedure Equivalent
```

Comment:
`.` after "procedure" is missing.

Reply:
Also link to procedure docs and add example